calculator.js

```diff
@@ -5,15 +5,50 @@ const exampleAdditionInput = {
   operation: 'add',
 }
 
+
 const calculateUserInput = function (error, promptInput) {
-  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+  if(isNaN(parseInt(promptInput.num1))) {
+    console.log(`${promptInput.num1} is not a number`);
+    return
+  } else if(isNaN(parseInt(promptInput.num2))) {
+    console.log(`${promptInput.num2} is not a number`)
+    return
+  }
+
+
+
+  switch(promptInput.operation){
+    case 'add':
+    case '+':
+      addNums(promptInput.num1, promptInput.num2)
+    break
+
+    case 'subtract':
+    case '-':
+      subtractNums(promptInput.num1, promptInput.num2)
```

Comment:
Yeaaahhh! This is a great way to write this switch statement! one note, we normally leave the `break;` indented on the same line as the the contents like so:
```suggestion
  switch(promptInput.operation){
    case 'add':
    case '+':
      addNums(promptInput.num1, promptInput.num2);
      break;

    case 'subtract':
    case '-':
      subtractNums(promptInput.num1, promptInput.num2);
```